### DISCUSSION TITLE
Taxa de $20 cobrado pela Husky

### DISCUSSION THREAD

**Discussion by brnpimentel** [DyegoCosta/trabalhando-remoto#201] [General]
Pessoal,

Quem está recebendo com a Husky, vocês estão recebendo com $20 a menos? Já recebi de duas empresas de fora (EUA) e nas duas foram retirados $20. Entrei em contato com a Husky, que diz que é a taxa que o banco que transferiu o dinheiro lá nos EUA cobra. 

Vocês estão passando por isso?

**Comment by heliomeiralins** [comment]
Já passei por isso em uma empresa que trabalhei parece que é verdade. Nas duas últimas empresas que trabalhei não está sendo cobrado.

Eu lembro que eu tirei o banco intermediário das informações achando que podia ser isso mas a taxa foi cobrada do mesmo jeito.

Abs

**Comment by brnpimentel** [comment]
Será que não estão usando o banco intermediário? No log do Swift não mostra nada desse banco intermediário.

Eu entrei em contato com o Tranferbank para conhecer a taxas e etc, e perguntei sobre essa taxa do banco de origem, eles falaram:

```
Realmente, isso ocorre no banco do exterior, entretanto tem uma forma de solicitar que não haja a cobrança dessa despesa. Porém o banco no exterior deve acatar!`
Somente você dando a instrução ao pagador no exterior (nossa equipe auxilia)
Entretanto não tem como ter total garantia, pois realmente depende do banco em que seu pagador utiliza no exterior
```

Então aparentemente tem um meio.

**Comment by daniloisr** [comment]
Já aconteceu comigo em alguns trabalhos e era 20$ também, chamavam de "wire transfer fee". Talvez vc consiga negociar mesmo com o seu pagador, pois essa taxa é custo dele e provavelmente não está no seu contrato.

**Comment by igorlvicente** [comment]
Já aconteceu isso comigo uma vez no Remessa Online.
Pelo que sei, essa taxa sempre é cobrada. O que muda é que algumas empresas pagam por ela (como a minha) e outras deixam que o recebedor pague.
Verifique com sua empresa qual o modo que eles usam.

**Comment by andreksu** [comment]
Oi pessoal, sim infelizmente há alguns bancos dos EUA (no qual as empresas usam) que cobram $20 por transferência. No caso, o ideal é vc conversar com a empresa e ver dela cobrir estes $20.
